apps/wing/src/cli.ts

```diff
@@ -144,6 +144,10 @@ async function main() {
     .argument("[entrypoint]", "program .w entrypoint")
     .option("-p, --port <port>", "specify port")
     .option("--no-open", "Do not open the Wing Console in the browser")
+    .option(
+      "-w, --watch <globs...>",
+      "Watch additional paths for changes. Supports globs and '!' for negations."
+    )
```

Comment:
Would love to add a test for this, but I don't see a good place to do so. The CLI currently mocks the console for its tests, and the console server doesn't seem to have any tests to add to. I'm open to ideas for useful testing.